modules/mukhin_i_numerical_integration/src/integration_methods.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2024 Mukhin Ivan
+
+#include "include/integration_methods.h"
+#include <cstdlib>
+
+double IntegrationMethods::integrate_trapezoid(
+        const std::function<double(double)> &f,
+        double x0, double xn, int n
+    ) {
+    n = static_cast<double>(n);
+    double step = std::abs(xn - x0) / n;
```

Comment:
Так как параметр `n` используется в дальнейшем в качестве количества итераций цикла, то лучше его привести к типу _double_ только при вычислении `step` 
```suggestion
    double step = std::abs(xn - x0) / static_cast<double>(n);
 ```

Reply:
Исправлено